### PR TITLE
Bad 638 update readthedocs

### PR DESCRIPTION
**Pull request by kathrynnapier**
Hi @keegansmith21 

With the ongoing transition of the BAD project to a service operated by OAPEN, we've updated the overview documentation (developed by OAPEN). Can you please take a look and review?

### CODE REVIEW

File: docs/bad_project/overview/background.md [hunk 1]
```diff
@@ -1,23 +1,19 @@
...
+Books are made available through a multitude of different platforms, and each has its own way of providing usage statistics. Using our Dashboard gives you a single point of synthesis of usage data from a wide range of [sources](dashboard_data.md), allowing you a consolidated view of usage. 
 
+We take care to bring together a synthesis of usage data of known provenance to allow you to compare data across your chosen time span. Learn more about how we collect and [process book usage data](how_works.md).
 
-## Partner Organisations
-This project is led by Principle Investigators Lucy Montgomery and Cameron Neylon from [Curtin University](https://www.curtin.edu.au/), Niels Stern and Ronald Snijder from the [OAPEN Foundation](https://oapen.org/), and Katherine Skinner (formerly [Educopia Institute](https://educopia.org/), now Research Lead at IOI). 
+## Who’s behind the Dashboard and how it is operated?
```
Comment (keegansmith21) [inline docs/bad_project/overview/background.md:11]:
```suggestion
## Who’s behind the Dashboard and how is it operated?
```

Comment (kathrynnapier) [reply docs/bad_project/overview/background.md:11]: @keegansmith21 I agree, I'll change it- but it may come back after review from Laura next week.

**Review by keegansmith21 — APPROVED** [review]
Thanks for making these changes Kathryn. Merge when you're ready :+1:

**Comment by kathrynnapier** [comment]
Thanks @keegansmith21 !

**Pull request merged**